.github/workflows/build-and-deploy.yml

```diff
@@ -15,15 +15,24 @@ jobs:
   deploy-docs:
     runs-on: ubuntu-latest
 
-    # Only run this job if there were changes in the docs folder
-    paths:
-      - ./docs/*
-    
```

Comment:
Turns out you can't use `paths` to conditionally run a job. (It only works on the workflow level, i.e. the top most level).